INSTALL.md

```diff
@@ -1,4 +1,38 @@
 # Installation
+This document provides important information on how to setup and install SBSCL and its optional dependencies.
+
+
+SBSCL can be build with maven. To build without running the tests use
+```
+mvn clean install -DskipTests
+```
+
+The tests require additional resources which have to be downloaded. These include the sbml-testsuite and the BiGG models.
+```
+source ./src/test/download_bigg_models.sh
+source ./src/test/download_sbml-test-suite.sh
+```
+which then can be run as part of the build step
+```
+mvn clean install
+```
+
+## SBML Test Runner
+The SBML Test Runner is a standalone desktop application with a graphical user interface (GUI) for controlling SBML-compatible applications and making them run test cases from the [SBML Test Suite](http://sbml.org/Software/SBML_Test_Suite). It is written in Java with the Eclipse SWT widget toolkit and can be used on macOS, Linux and Windows.
```

Comment:
Delete: `It is written in Java with the Eclipse SWT widget toolkit and can be used on macOS, Linux and Windows.`

Reply:
Updated!